src/Microsoft.AspNetCore.Server.Kestrel.Core/Adapter/Internal/LoggingStream.cs

```diff
@@ -131,87 +131,5 @@ private void Log(string method, int count, byte[] buffer, int offset)
 
             _logger.LogDebug(builder.ToString());
         }
-
-#if NET46
-        // The below APM methods call the underlying Read/WriteAsync methods which will still be logged.
-        public override IAsyncResult BeginRead(byte[] buffer, int offset, int count, AsyncCallback callback, object state)
```

Comment:
These APIs exist in netcoreapp2.0. You should remove the ifdef instead of deleting this